test_utils/basetest.py

```diff
@@ -12,6 +13,9 @@ def create_credentials(self):
         self.azure_credential = DefaultAzureCredential()
         self.subscription_id = os.environ["AZURE_SUBSCRIPTION_ID"]
         self.resourcegroup_location = os.environ["AZURE_DEFAULT_REGION"]
+        self.sumo_access_id = os.environ["SUMO_ACCESS_ID"]
+        self.sumo_access_key = os.environ["SUMO_ACCESS_KEY"]
+        self.sumo_deployment = os.environ["SUMO_DEPLOYMENT"]
```

Comment:
self.sumologic_cli = SumoLogic(os.environ["SUMO_ACCESS_ID"], os.environ["SUMO_ACCESS_KEY"], self.api_endpoint(os.environ["SUMO_DEPLOYMENT"])) 
no need to save access id and access key locally we won't need them later once sumologic_cli gets created.

Reply:
updated